Rubberduck.VBEEditor/SafeComWrappers/VBA/CodeModule.cs

```diff
@@ -153,9 +153,7 @@ public int GetProcCountLines(string procName, ProcKind procKind)
 
         public string GetProcOfLine(int line)
         {
-            if (IsWrappingNullReference) return string.Empty;
-            vbext_ProcKind procKind;
-            return Target.get_ProcOfLine(line, out procKind);
+            return IsWrappingNullReference ? string.Empty : Target.get_ProcOfLine(line, out _);
```

Comment:
This is beyond the scope of this PR but I should point out that this method basically discards the `vbext_ProcKind` that's returned. Is there any value in returning this as an optional `out` parameter back to the callees of the `GetProcOfLine`, making it their choice whether they care about it or not?

Reply:
@bclothier there would if the caller used the VBIDE API, but the VBIDE API wrapper types are rather deep, ...it's possible that future calls to `GetProcOfLine` might possibly need the parameter, but somewhat unlikely. IOW if it's not needed by now... YAGNI ;-)